src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -679,6 +681,7 @@ export class WebviewElement extends Disposable implements IWebview, WebviewFindD
 
 	protected handleFocusChange(isFocused: boolean): void {
 		this._focused = isFocused;
+		this._focusedContextKey?.set(isFocused);
```

Comment:
@amunger Done. Thanks for the suggestion, done
This should handle all webviews.